packages/commerce-sdk-react/package.json

```diff
@@ -17,6 +21,7 @@
         "url": "https://github.com/SalesforceCommerceCloud/pwa-kit/issues"
       },
     "scripts": {
+      "prepare": "npm run build",
```

Comment:
why do we need this?

Reply:
I added that for consistency since I noticed that our other sdk package.jsons all have a `prepare` script as well.